mvi/src/main/kotlin/com/adidas/mvi/ReducerExt.kt

```diff
@@ -0,0 +1,9 @@
+package com.adidas.mvi
+
+@Suppress("UNCHECKED_CAST")
+public inline fun <reified TView : Any> Reducer<*, *>.requireView(): TView =
```

Comment:
Can you do it like public inline fun <TState, reified TView> Reducer<TState, *>.requireView?